applications/matter_weather_station/README.rst

```diff
@@ -70,6 +67,16 @@ For details, see the `Testing`_ section.
 
 .. _matter_weather_station_app_build_types:
 
+Matter weather station product certification
+============================================
+
+The Matter weather station application running on Thingy:53 participated in Matter Specification Validation Event (SVE) and successfully passed all required test cases to be considered as Matter 1.0 compliant device.
```

Comment:
OK, so I would move this entire section to the introduction, line 15, maybe to a note, and then link to this section. Reason being that certification status is not part of the application, only certification files are, so it makes sense to separate information about the certification status and highlight it in the incipit.

```suggestion
The Matter weather station application running on Thingy:53 participated in Matter Specification Validation Event (SVE) and successfully passed all required test cases to be considered as Matter 1.0 compliant device.
See `Matter weather station certification files`_ for more information.
```